### ISSUE TITLE
pnpm/nextjs build hangs again if next outputStandalone is used

### ISSUE THREAD

**Issue by sdorra** [vercel/turborepo#1481]
### What version of Turborepo are you using?

1.3.1

### What package manager are you using / does the bug impact?

pnpm

### What operating system are you using?

Mac

### Describe the Bug

Turbo hangs on next build, if pnpm and next [outputStandalone](https://nextjs.org/docs/advanced-features/output-file-tracing) is used.
The output shows the turbo build summary, but the process never ends.

The bug was introduced with version 1.2.6, fixed in 1.2.12 (#1265) and reintroduced in 1.3.0.

I've updated the repository (https://github.com/sdorra/turbo-next-standalone-issue) to turbo 1.3.1.

### Expected Behavior

The build process ends.

### To Reproduce

```
git clone https://github.com/sdorra/turbo-next-standalone-issue
cd turbo-next-standalone-issue
pnpm install
pnpm run build
```

**Comment by tknickman** [comment]
Your next config is slightly incorrect which seems to be causing some undefined build behavior. 

Only `outputFileTracingRoot` is experimental. Not `outputStandalone`.

So instead of:
```
const nextConfig = {
  experimental: {
    outputStandalone: true,
    outputFileTracingRoot: path.join(__dirname, "../../"),
  },
};
```

Try:
```
const nextConfig = {
  outputStandalone: true,
  experimental: {
    outputFileTracingRoot: path.join(__dirname, "../../"),
  },
};
```

This fixes the issue in your reproduction repo.

**Comment by tknickman** [comment]
Closing for now, but please feel free to reopen if you continue to see the issue!

**Comment by sdorra** [comment]
@tknickman the option is experimental in [next.js 12.1.6](https://github.com/vercel/next.js/blob/v12.1.6/packages/next/server/config-shared.ts#L115). Moving the option outside the experimental object fixes the issue, because it disables the standalone output.

I've updated the [sample repo](https://github.com/sdorra/turbo-next-standalone-issue) to the [latest next.js version (12.2.2) and updated the configuration](https://github.com/sdorra/turbo-next-standalone-issue/commit/8579adfe4f5255e1e6ba33447a5c58409649b0c9). The issue still exists.

It seems that i've not privileged to reopen the issue.

**Comment by azzlack** [comment]
I have the same issue when using pnpm, worked fine with yarn

**Comment by azzlack** [comment]
@tknickman please reopen :-)

**Comment by baptisteArno** [comment]
I have the exact same issue with next@12.2.4, turbo@1.4.2 and pnpm@7.8.0

**Comment by tuahear** [comment]
I'm having the same issue

**Comment by tuahear** [comment]
I just updated to the latest next@12.2.5 and the problem is gone after several tests. You may wanna give it a try. @azzlack @baptisteArno 😄

**Comment by baptisteArno** [comment]
Nope, doesn't seem to solve it for me! That's strange??

**Comment by tuahear** [comment]
Weird. I'm btw using... node@16.16.0, pnpm@7.9.1, turbo@1.4.3, next@12.2.5

**Comment by azzlack** [comment]
Did not work for me either @baptisteArno @tuahear . Running node@16.17.0, pnpm@7.9.3, turbo@1.4.3, next@12.2.5

**Comment by antoniodinovi** [comment]
> Did not work for me either @baptisteArno @tuahear . Running node@16.17.0, pnpm@7.9.3, turbo@1.4.3, next@12.2.5.

Same configuration same behavior. I run it on gitlab ci

**Comment by baptisteArno** [comment]
Why is it ignored @tknickman @jaredpalmer ?

**Comment by tknickman** [comment]
Apologies @baptisteArno, this one slipped through the cracks. Can you test this again with the latest turbo canary? We recently fixed an issue related to hanging when using next in standalone mode.

**Comment by antoniodinovi** [comment]
> Apologies @baptisteArno, this one slipped through the cracks. Can you test this again with the latest turbo canary? We recently fixed an issue (#1719) related to hanging when using next in standalone mode.

Great!!!
Using ` "turbo": "1.4.4-canary.0"` build not hangs anymore
